composer.json

```diff
@@ -38,7 +38,7 @@
         "openeuropa/oe_contact_forms": "~1.1",
         "openeuropa/oe_content": "^2.8.0",
         "openeuropa/oe_corporate_blocks": "^4.4",
-        "openeuropa/oe_media": "^1.14",
+        "openeuropa/oe_media": "^1.16",
```

Comment:
why do you need to bump this up? what fails ?

Reply:
Because is the oe_bootstrap_theme 1.x required version: https://github.com/openeuropa/oe_bootstrap_theme/blob/1.x/composer.json

So as I removed there, I updated in the other components.

Reply:
No, that was a mistake on my side, so I'm glad we removed :)
But please drop the commit that bumbs this